js/nightly.js

```diff
@@ -11,4 +11,19 @@ $().ready(function() {
     $('#builds a').click(function() {
         _gaq.push(['_trackEvent', 'Firefox Downloads', 'download click', 'Firefox Nightly']);
     });
+
+    // Added for Bug 1213905
+    // Spotlight Activity Tag: Mozilla (6247) | Mozilla B2G Activity Tag 2 (52162) | Mozilla Spots (4669)
+    $('.b2gdroid a').click(function(event) {
+        var ftRand = Math.random() + "";
+        var num = ftRand * 1000000000000000000;
+        var ftGoalTagPix52162 = new Image();
+        var href = $(this).attr('href');
+
+        ftGoalTagPix52162.onload = function() { location.href = href; };
+        ftGoalTagPix52162.src = "http://servedby.flashtalking.com/spot/8/6247;52162;4669/?spotName=Mozilla_B2G_Activity_Tag_2&cachebuster="+num;
+
+        event.preventDefault();
+        return false;
```

Comment:
You shouldn't need `return false;` as you're already using `event.preventDefault()`;